automation/taskcluster/decision_task_release.py

```diff
@@ -0,0 +1,209 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Decision task for releases
+"""
+
+import argparse
+import json
+import os
+import taskcluster
+import yaml
+
+import lib.tasks
+
+TASK_ID = os.environ.get('TASK_ID')
+
+BUILDER = lib.tasks.TaskBuilder(
+    task_id=TASK_ID,
+    repo_url=os.environ.get('GITHUB_HEAD_REPO_URL'),
+    branch=os.environ.get('GITHUB_HEAD_BRANCH'),
```

Comment:
Nit: change this var to be the tag instead.

Reply:
I just realized I don't pass these along in the `.taskcluster.yml`. Wasn't I supposed to have it defined here - https://github.com/mozilla-mobile/android-components/pull/803/files#diff-ac0229d1171c0983b27003af4c7441a5R75 ?